### PR TITLE
Staging arturas

### PR DESCRIPTION
**Pull request by artciukas**
Updated main file with printing info's and user interface.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -1,33 +1,48 @@
...
 
+user_name = input("Plese enter user name: ")
+print("*************")
+print(f"Welcome {user_name}")
+print("*************")
 
 while True:
 
-    choice = input("1 - enter task\n2 - find task\n3 - update task\n4 - delete task\n0 - exit\n")
+    choice = input("1 - enter task\n2 - find task\n3 - update by task title\n4 - delete task by name\n0 - exit\n")
```
Comment (minrudev) [inline main.py:11]: Check for the right type

File: main.py [hunk 2]
```diff
@@ -1,33 +1,48 @@
...
 
     match choice:
```
Comment (minrudev) [inline main.py:13]: Make sure that your partner use python > 3.10

**Review by minrudev — CHANGES_REQUESTED** [review]

File: main.py (hunk 1 above)
Comment (LaimonasAR) [reply main.py:11]: "match choice" requires string, and string is received from "input".
If string does not match listed values, loop starts from the beginning.

File: main.py (hunk 2 above)
Comment (LaimonasAR) [reply main.py:13]: updated README.md

**Review by LaimonasAR — APPROVED** [review]

**Pull request merged**